tsdoc.json

```diff
@@ -1,5 +1,6 @@
 {
     "$schema": "https://developer.microsoft.com/json-schemas/tsdoc/v0/tsdoc.schema.json",
+    // Correct url: https://developer.microsoft.com/en-us/json-schemas/tsdoc/v0/tsdoc.schema.json
```

Comment:
Not sure why this comment is here, but JSON doesn't support comments

Reply:
I put that there because the url doesn't work and you can't change it: https://github.com/microsoft/tsdoc/issues/253
I removed, the tsdoc used jsonc so the comments didn't break anything but looks like the issue might be solved soon.

(Also updated the description)